.github/workflows/main.yml

```diff
@@ -41,6 +41,9 @@ jobs:
   functional-test:
     name: Functional test with LXD
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        juju: ["3/stable", 2.9/stable]
```

Comment:
this is master, I think it would be OK to just test against `3/stable` and drop 2.9

Reply:
fine by me of course... are there any other juju versions we'd like to matrix?